templates/index.html

```diff
@@ -9,10 +9,10 @@
 <section class="row row-hero">
 	<div class="inner-wrapper overflow-hidden">
 		<div class="row-hero__text row-hero__col eight-col margin-top--30 padding-top--30">
-			<h1><img src="https://assets.ubuntu.com/v1/33b96a7f-maas-logo.svg" alt="" class="row-hero__logo" /> Metal as a Service</h1><h1>Say hello to your physical cloud</h1>
-			<h3 class="padding-bottom--30">Total automation of your physical servers for amazing data centre operational efficiency. On premise, open source and supported.</h3>
+			<h1><img src="https://assets.ubuntu.com/v1/33b96a7f-maas-logo.svg" alt="" class="row-hero__logo" itemprop="image" /><span itemprop="name">Metal as a Service</span><br/>Say hello to your physical cloud</h1>
+			<h3 class="padding-bottom--30" itemprop="description">Total automation of your physical servers for amazing data centre operational efficiency. On premise, open source and supported.</h3>
```

Comment:
Can you make this paragraph tag with style to match the design. Semantically, it should be a h2 if it's going to be a header but it to long and does not title any content. Therefore it's intro text.